misc/python/materialize/checks/scenarios.py

```diff
@@ -70,15 +70,17 @@ def run(self) -> None:
         # Configure implicitly for cloud scenarios
         if not isinstance(actions[0], StartMz):
             actions.insert(0, ConfigureMz(self))
-        for action in actions:
-            action.execute(self.executor)
-            action.join(self.executor)
+
+        for id, action in enumerate(actions):
```

Comment:
Nit (optional): I would prefer `index` instead of `id` in such a loop header.

Reply:
done